backend/hyperglosae/src/updates/document.js

```diff
@@ -0,0 +1,22 @@
+function(doc, req) {
+    const body = JSON.parse(req.body);
+    if (!doc) {
+        doc = Object.assign({
+            _id: req.id,
+            _rev: req.rev,
+            history: [{
+                user: req.userCtx.name,
+                date: new Date().toISOString(),
+            }]
+        }, body);
```

Comment:
Are you sure CouchDB interpreter cannot handle more "modern" way to express this?

With spare operators (instead of `Object.assign()`) this would be expressed as:

```js
doc = {
  ...body ,
  _id: req.id,
  _rev: req.rev,
  history: [{
    user: req.userCtx.name,
    date: new Date().toISOString(),
}];
```